modules/azurerm_netapp/variables.tf

```diff
@@ -23,6 +23,10 @@ variable "subnet_id" {
   description = "Azure subnet id for Azure NetApp Files"
 }
 
+variable "network_features" {
+  description = "Indicates which network feature to use, accepted values are Basic or Standard, it defaults to Basic if not defined."
+}
```

Comment:
Wow have a lot of non-existent `type` fields in this file. If we don't want to force it to something we can add 
```yaml
  type = null
```
or 
```yaml
  type = any
```
This is just a general comment not for this issue, but need to start making sure we know the types of our variables in the future.

Reply:
Done. Added `type = string` for the new variable. I'll create a separate new ticket to make similar updates to the other old variables.